Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -370,6 +370,7 @@ int RGroupDecomposition::add(const ROMol &inmol) {
   }
 
   const bool prune = true;
+  // in case the value ends up being changed in a future version of the code:
```

Comment:
I really don't think this is useful and is confusing when reviewing the code.  I don't mind the const bool prune, but the if statement below screams to be gone.  I vote for getting rid of it and making prune the default.  If we want to be able to change it in the future, make it a member variable, but this code is complex enough removing dead code is a good thing :)